cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaBridgeClusterTest.java

```diff
@@ -1234,12 +1233,6 @@ public void testProbeConfiguration()   {
         assertThat(cont.getReadinessProbe().getTimeoutSeconds(), is(32));
     }
 
-    @ParallelTest
-    public void testJaegerTracingConfiguration() {
-        testTracingConfiguration(JaegerTracing.TYPE_JAEGER);
-    }
-
-    @ParallelTest
```

Comment:
Did you delete one of these `@ParallelTest` annotations by mistake?

Reply:
:-o .... good catch! Yes that was a mistake!